bodhi/server/scripts/approve_testing.py

```diff
@@ -108,8 +109,8 @@ def main(argv=sys.argv):
                 notifications.publish(
                     topic='update.requirements_met.stable',
                     msg=dict(update=update))
+                db.commit()
```

Comment:
Why not just slide this ```commit()``` left a notch and get rid of the first ```commit()```?

Reply:
My intention was to prevent unnecessary db.commit() calls(performance). For example, when query returns 100 updates, and only 2 will need call to db.commit(), 98 other will be unnecessary. But if you recommend db.comment() after each "for" iteration, I can change it.

Reply:
Cool, that sounds like a fine justification to me.